code/game/objects/items/control_wand.dm

```diff
@@ -8,7 +8,7 @@
 	item_state = "electronic"
 	icon = 'icons/obj/device.dmi'
 	name = "control wand"
-	desc = "Remotely controls airlocks."
+	desc = "Remotely controls airlocks. Current mode: open doors"
```

Comment:
```suggestion
	desc = "Remotely controls airlocks."
```

`examine` already adds `desc` further up, so it would only be duplicated

Reply:
Oh yeah, forgot about this....